src/injectHighlightWrappers.ts

```diff
@@ -319,13 +321,13 @@ function createWrapper(options: any) {
     span.setAttribute(TIMESTAMP_ATTR, options.timestamp);
   }
   if (options.id) {
-    span.setAttribute('data-id', options.id);
+    span.setAttribute(DATA_ID_ATTR, options.id);
   }
   return span;
 }
 
 function isHighlight(el: any): el is HTMLElement {
-  return el && el.nodeType === NODE_TYPE.ELEMENT_NODE && el.hasAttribute(DATA_ATTR);
+  return el && el.nodeType === NODE_TYPE.ELEMENT_NODE && el.matches(DATA_ATTR_SELECTOR);
```

Comment:
is this actually better in some way?

Reply:
not especially.  `matches` will accept any CSS selector but there's not any reason to use it here since we're only testing for the existence of an attribute